README.md

```diff
@@ -68,7 +105,7 @@ enum Environment {
 Providing the mapping type `environment : Environment` in the mapping file, and the string `.Production` in the plist, the property in your configuration class will be as follows:
 
 ```
-  static let environment: Environment = .Production
```

Comment:
As of Donal's commit the "static" prefix was removed so it isn't generated anymore. Should I re-add it?